### ISSUE TITLE
 About the accuracy of pointcnn for Modelnet40

### ISSUE THREAD

**Issue by LiDaiY** [Guajevilla/PointNet#2]
Thank you for your implementation of PointCNN, it's very concise and easy to understand. Have you trained on classified tasks using PointCNN? It is very appreciated if you would share the results you've gotten.

I have trained the PointCNN(use this repo's code) for classification task(ModelNet40), 
the parameters and results are as follows:

`Namespace(batch_size=24, batch_size_test=24, decay_rate=0.0001, epoch=200, gpu='0', learning_rate=0.001, log_dir=None, model='pointcnn_cls_simplied', normal=False, num_point=1024, optimizer='Adam')`

`Epoch 200 (200/200):`
`Train Instance Accuracy: 0.913678`
`Test Instance Accuracy: 0.912136, Class Accuracy: 0.882880`
`Best Instance Accuracy: 0.917799, Class Accuracy: 0.885735`

**Comment by heexu976** [comment]
hello can I read ur training code?
